cmd/edit/machinepool/cmd_test.go

```diff
@@ -1,33 +1,221 @@
+				// First get
+				t.ApiServer.AppendHandlers(RespondWithJSON(http.StatusOK, mpResponse))
+				// Edit
+				t.ApiServer.AppendHandlers(RespondWithJSON(http.StatusOK, ""))
+				t.SetCluster(clusterId, mockClassicClusterReady)
+				args := NewEditMachinepoolUserOptions()
+				args.machinepool = nodePoolId
+				runner := EditMachinePoolRunner(args)
+				cmd := NewEditMachinePoolCommand()
+				err := cmd.Flag("cluster").Value.Set(clusterId)
+				Expect(err).ToNot(HaveOccurred())
+				err = cmd.Flags().Set("labels", "test=test")
+				Expect(err).ToNot(HaveOccurred())
+				err = runner(context.Background(), t.RosaRuntime, cmd,
+					[]string{"--machinepool", nodePoolId, "--min-replicas", "2", "--enable-autoscaling", "true", "-y"})
+				Expect(err).ToNot(HaveOccurred())
+			})
+			It("Machinepool ID passed in without flag in random location", func() {
+				t.ApiServer.AppendHandlers(RespondWithJSON(http.StatusOK, classicClusterReady))
+				// First get
+				t.ApiServer.AppendHandlers(RespondWithJSON(http.StatusOK, mpResponse))
+				// Edit
+				t.ApiServer.AppendHandlers(RespondWithJSON(http.StatusOK, ""))
+				t.SetCluster(clusterId, mockClassicClusterReady)
+				args := NewEditMachinepoolUserOptions()
+				args.machinepool = nodePoolId
+				runner := EditMachinePoolRunner(args)
+				cmd := NewEditMachinePoolCommand()
+				err := cmd.Flag("cluster").Value.Set(clusterId)
+				Expect(err).ToNot(HaveOccurred())
+				err = cmd.Flags().Set("labels", "test=test")
+				Expect(err).ToNot(HaveOccurred())
+				err = runner(context.Background(), t.RosaRuntime, cmd,
+					[]string{"--min-replicas", "2", nodePoolId, "--enable-autoscaling", "true", "-y"})
+				Expect(err).ToNot(HaveOccurred())
+			})
+		})
+
+		Describe("Nodepools", func() {
+			It("Able to edit nodepool with no issues", func() {
+				t.ApiServer.AppendHandlers(RespondWithJSON(http.StatusOK, hypershiftClusterReady))
+				// First get
+				t.ApiServer.AppendHandlers(RespondWithJSON(http.StatusOK, nodePoolResponse))
+				// Edit
+				t.ApiServer.AppendHandlers(RespondWithJSON(http.StatusOK, ""))
+				t.SetCluster(clusterId, mockClusterReady)
+				args := NewEditMachinepoolUserOptions()
+				args.machinepool = nodePoolId
+				runner := EditMachinePoolRunner(args)
+				cmd := NewEditMachinePoolCommand()
+				err := cmd.Flags().Set("enable-autoscaling", "true")
+				Expect(err).ToNot(HaveOccurred())
+				err = cmd.Flags().Set("labels", "test=test")
+				Expect(err).ToNot(HaveOccurred())
+				err = cmd.Flag("cluster").Value.Set(clusterId)
+				Expect(err).ToNot(HaveOccurred())
+				err = cmd.Flags().Set("min-replicas", "2")
+				Expect(err).ToNot(HaveOccurred())
+				err = cmd.Flags().Set("max-replicas", "10")
+				Expect(err).ToNot(HaveOccurred())
+				err = cmd.Flag("yes").Value.Set("true")
+				Expect(err).ToNot(HaveOccurred())
+				err = runner(context.Background(), t.RosaRuntime, cmd,
+					[]string{"--machinepool", nodePoolId})
+				Expect(err).ToNot(HaveOccurred())
```

Comment:
I was wondering if this and code like this wouldn't be more readable like this:
```suggestion
				Expect(cmd.Flags().Set("enable-autoscaling", "true")).To(Succeed())
				Expect(cmd.Flags().Set("labels", "test=test")).To(Succeed())
				Expect(cmd.Flag("cluster").Value.Set(clusterId)).To(Succeed())
				Expect(cmd.Flags().Set("min-replicas", "2")).To(Succeed())
				Expect(cmd.Flags().Set("max-replicas", "10")).To(Succeed())
				Expect(cmd.Flag("yes").Value.Set("true")).To(Succeed())
				Expect(runner(context.Background(), t.RosaRuntime, cmd,
					[]string{"--machinepool", nodePoolId})).To(Succeed())
```
WDYT?

Reply:
:open_mouth: havent thought of this, let me try it out